src/sql/platform/restore/browser/restoreServiceImpl.ts

```diff
@@ -279,7 +279,7 @@ export class RestoreDialogController implements IRestoreDialogController {
 	}
 
 	private handleOnClose(): void {
-		this._connectionService.disconnect(this._ownerUri);
+		this._connectionService.disconnect(this._ownerUri).catch((e) => this._logService.error(e));
```

Comment:
>this._logService.error(e) [](start = 66, length = 25)

Does this print out the stack trace as well? Or at least something to identify where the error came from?

Reply:
The .error call should print out the stacktrace.

Reply:
Yup, just tested it and it does print out the full stack trace.

![image](https://user-images.githubusercontent.com/28519865/68170529-07b9ca00-ff25-11e9-943b-3df5733d763d.png)